defaults.env

```diff
@@ -0,0 +1,22 @@
+# DEFAULTS.ENV is a Docker .env file for configuring cores aspects of Janeway
+#
+# here are the Docker-Compose env file rules: https://docs.docker.com/compose/env-file/
+# here are the Lando env file rules: https://docs.lando.dev/config/env.html
+#
+# NOTE that Lando allows more than one env file to be used, which lets you
+# override certain variables while leaving defaults in place. Consider using an
+# override instead of modifying the defaults here.
+#
+# To override the defaults set below, copy the local.env.example file to
+# local.env and then set the values in local.env appropriately.
+#
+###############################################################################
+# Database
+
+DB_VENDOR=postgres
+DB_NAME=janeway
+DB_USER=postgres
+DB_PASSWORD=
+DB_HOST=db.janeway.internal
+DB_PORT=5432
```

Comment:
Should live in the `dockerfiles` directory. Probably worth renaming it so it is clear this overrides the environment for lando and not regular docker-compose.yml (e.g. `lando.env`)

Reply:
Moved to dockerfiles.